Bootstrap fails on fresh clone

**Describe the bug**
Running `yarn bootsrap --all` fails with the following error:
```bash
lerna ERR! yarn run prepare exited 2 in '@storybook/react-native'
lerna ERR! yarn run prepare stdout:
$ node ../../scripts/prepare.js
src/preview/index.tsx(193,63): error TS2558: Expected 0 type arguments, but got 1.
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/Shared/theme.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/Shared/text.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/StoryListView/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/StoryView/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/navigation/button.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/addons/list.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/addons/wrapper.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/addons/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/panel.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/navigation/constants.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/navigation/bar.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/navigation/visibility-button.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/navigation/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/absolute-positioned-keyboard-aware-view.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/animation.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/components/OnDeviceUI/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/preview/index.js
TSFILE: /Users/vincepicone/Programming/Work/storybook-react-native/app/react-native/dist/index.js
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.

lerna ERR! yarn run prepare stderr:
ERR! FAILED (ts) :  
ERR! FAILED to compile ts: @storybook/react-native@6.0.0-alpha.0 
error Command failed with exit code 2.

lerna ERR! yarn run prepare exited 2 in '@storybook/react-native'
error Command failed with exit code 2.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

**To Reproduce**
1. clone the repository
2. run `yarn bootstrap --all` (per the contribution docs)

**Expected behavior**
Project bootstraps successfully

**System:**
```
  System:
    OS: macOS 10.15.5
    CPU: (8) x64 Intel(R) Core(TM) i7-7820HQ CPU @ 2.90GHz
  Binaries:
    Node: 12.18.1 - ~/.nvm/versions/node/v12.18.1/bin/node
    Yarn: 1.22.4 - ~/.yarn/bin/yarn
    npm: 6.14.5 - ~/.nvm/versions/node/v12.18.1/bin/npm
  Browsers:
    Chrome: 80.0.3987.132
    Firefox: 74.0
    Safari: 13.1.1
  npmPackages:
    @storybook/eslint-config-storybook: ^2.0.0 => 2.1.0 
    @storybook/linter-config: ^2.0.0 => 2.1.0 
```

--- a/app/react-native/src/preview/index.tsx
+++ b/app/react-native/src/preview/index.tsx
@@ -190,7 +190,7 @@ More info: https://github.com/storybookjs/storybook/blob/next/MIGRATION.md#react
       try {
         let value = this._asyncStorageStoryId;
         if (!value && this._asyncStorage) {
-          value = JSON.parse(await this._asyncStorage.getItem<string>(STORAGE_KEY));
+          value = JSON.parse(await this._asyncStorage.getItem(STORAGE_KEY));
           this._asyncStorageStoryId = value;
         }
 

